lesson07/assignment/vue-project/src/components/Grid.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <!-- Container -->
+  <b-container class="section-grid grid">
+    <h2 class="pt-5 display-4 text-muted">{{ header }}</h2>
+    <p class="lead mb-4 text-muted">{{ subtitle }}</p>
+
+    <!-- Container / Grid Columns -->
+    <b-container>
+      <b-row>
+        <!-- Loop thru an array of contents and dynamically generate columns with content  -->
```

Comment:
I like how you took things a step further and implemented a for loop for your grid, as well!